### DISCUSSION TITLE
How to return a list field?

### DISCUSSION THREAD

**Discussion by saeedesmaili** [fastapi/fastapi#7426] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

I'm using a model like:
```
class Question(BaseModel):
    id: int
    title: str = Field(..., min_length=3, max_length=50)
    answer_true: str = Field(..., min_length=3, max_length=50)
    answer_false: list
    category_id: int
```
And trying to get the `questions` using the following function:
```
def get(id: int):
    query = questions.select().where(id == questions.c.id)
    return database.fetch_one(query=query)

@router.get("/{id}/", response_model=QuestionDB)
def read_question(id: int = Path(..., gt=0),):
    question = get(id)
    if not question:
        raise HTTPException(status_code=404, detail="question not found")
    return question
```
And here is the data that has been stored in the database:
![image](https://user-images.githubusercontent.com/13351748/82663777-91bd4c00-9c45-11ea-8481-18de346f8754.png)

But it can't return the `list` field (`answer_false`) correctly and this field's values are being returned as characters:
![image](https://user-images.githubusercontent.com/13351748/82663959-f5e01000-9c45-11ea-95c5-f2289aa44c7e.png)
What am I doing wrong and how should I fix this?

**Comment by saeedesmaili** [comment, marked as answer]
It was because of my sqlalchemy config. I Removed `dimensions` in the table's config it the problem got fixed:

```
questions = Table(
    "questions",
    metadata,
    Column("id", Integer, primary_key=True),
    Column("title", String(50)),
    Column("answer_true", String(50)),
    Column("answer_false", ARRAY(String)), ## adding dimension will cause the list to not work in get requests
    Column("created_date", DateTime, default=func.now(), nullable=False),
    Column("category_id", Integer, ForeignKey("categories.id")),
)
```

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue :+1:
